app/src/main/java/rightmesh/left/io/gpstracker/Constants.java

```diff
@@ -0,0 +1,9 @@
+package rightmesh.left.io.gpstracker;
+
+public class Constants {
+    public static final int MESH_PORT = 5001;
+    public static final String SUPER_PEER_ID = "0x656284abf20af4192d86f2f6f3e7ce04e5718302";
+
+    // TODO: fill in with your SuperPeer URL
+    public static final String SUPER_PEER_URL = "192.168.3.151";
```

Comment:
If this isn't clearly visible in the `MainActivity` anymore, it might be worth making sure it's documented in the README (if it isn't already) that you need to find it and change it.

Reply:
I'll do in the next version